src/app/signing/signing.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="container py-5 center-children vertical-center">
+  <div class="row">
+    <h2>Sending Message to friend.</h2>
+  </div>
+  <div class="row">
+    <textarea rows="8" class="form-control" id="signing-box" autofocus [(ngModel)]='signingBoxText'></textarea>
+  </div>
+
+  <div class="row" id="options">
+    <div class="btn-toolbar" role="toolbar" aria-label="Toolbar for formatting text"></div>
+
+    <div class="btn-group mr-2" role="group" aria-label="Font group">
+      <button type="button" class="btn btn-primary">Font</button>
+      <button type="button" class="btn btn-primary dropdown-toggle dropdown-toggle-split" data-toggle="dropdown"
+        aria-haspopup="true" aria-expanded="false">
+        <span class="sr-only">Toggle Dropdown</span>
+      </button>
+      <div class="dropdown-menu">
+        <a class="dropdown-item" href="signing">Arial</a>
+        <a class="dropdown-item" href="signing">Comic Sans</a>
+        <a class="dropdown-item" href="signing">Times New Roman</a>
+      </div>
+    </div>
+
+    <button type="button" class="btn btn-outline-dark mr-2" id="button-bold">
+      bold
+    </button>
+    <button type="button" class="btn btn-outline-dark mr-2" id="button-italics">
+      italics
+    </button>
+
+    <div class="btn-group mr-2" role="group" aria-label="Font group">
+      <div>
+        <input type="color" id="colorpicker mr-2" name="colorpicker" value="#ff0000" >
+      </div>
+      <div>
+        <input type="color" id="colorpicker mr-2" name="colorpicker" value="#00ff00">
+      </div>
+    </div>
+    <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#exampleModalCenter">
+      Submit
+    </button>
+  </div>
+  <div class="row">
+    <h3>Preview:</h3>
+  </div>
+  <textarea 
+  class="shadow"
+  id="signing-box" 
+  autofocus
+  ></textarea>
+  <div class="row">
```

Comment:
Did you get around to fixing the test display inside of this div so it has a fixed width?

- Look in the Markdown library for solutions
- Use some css to manually limit the sizes of the paragraph tags inside the div

Reply:
Not yet, that will be fixed on subsequent merges